yaml/csi-driver/edge/hpe-csi-k8s-1.29.yaml

```diff
@@ -812,7 +812,7 @@ spec:
             - name: socket-dir
               mountPath: /var/lib/csi/sockets/pluginproxy/
         - name: csi-volume-group-provisioner
-          image: quay.io/hpestorage/volume-group-provisioner:v1.0.6
+          image: quay.io/hpestorage/volume-group-provisioner:v1.0.8-beta
```

Comment:
Volume Group Provisioner and Snapshotter should be in lock-step versioning, why is this drifting?

Reply:
This is fixed as commented above.